src/openrct2/core/JobPool.cpp

```diff
@@ -0,0 +1,103 @@
+#include "JobPool.hpp"
+
+JobPool::JobPool()
+    : _shouldStop(false),
+    _processing(0)
```

Comment:
They are specified in different order, fix either here or in the class declaration.